Makefile

```diff
@@ -0,0 +1,85 @@
+VERSION ?= 0.0.0
+GOOS ?= darwin
+GOARCH ?= amd64
+ifeq ($(GOOS),windows)
+	EXT = .exe
+else
+	EXT = 
+endif
+OUTPUT ?= soracom/dist/$(VERSION)/soracom_$(VERSION)_$(GOOS)_$(GOARCH)$(EXT)
+
+GOCYCLO_OVER ?= 23
+
+.PHONY: help
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+.DEFAULT_GOAL := help
+
+install-dev: ## Install dev dependencies
+	@echo 'Installing dependencies for development'
+	go get -u \
+		golang.org/x/lint/golint \
+		github.com/fzipp/gocyclo/cmd/gocyclo
+.PHONY:install-dev
+
+install: ## Install dependencies
```

Comment:
How about using `install-deps` instead? Some people might think `install` installs `soracom-cli` itself to somewhere.